src/DataValidation.py

```diff
@@ -259,6 +259,23 @@ def checkStartingItemsForValidItemsAndCategories():
                     if len([item for item in DataValidation.item_table if "category" in item and category_name in item["category"]]) == 0:
                         raise ValidationError("Item category %s is set as a starting item category, but is misspelled or is not defined on any items." % (category_name))
 
+    @staticmethod
+    def checkStartingItemsForBadSyntax():
+        if "starting_items" not in DataValidation.game_table:
+            return
+        
+        starting_items = DataValidation.game_table["starting_items"]
+
+        for starting_block in starting_items:
+            if type(starting_block) is not dict or len(starting_block.keys()) == 0:
+                raise ValidationError("One of your starting item definitions is not a valid dictionary.\n   Each definition must be inside {}, as demonstrated in the Manual documentation.")
+                
+            valid_keys = ["items", "item_categories", "random", "if_previous_item"]

```

Comment:
looks good But I did mark in schema _comment as a valid key to write comments, with this it would be marked as invalid